schema.graphql

```diff
@@ -353,6 +353,7 @@ type Loan @entity {
     amountRepaidBtc: Float
     height: Height!
     timestamp: DateTime!
+    day: DateTime!
```

Comment:
I'd suggest using `date` instead of `day`. 
Typically, I would expect a field called `day` to either contain the numeric value of the day of the week (0-6), or its name (Monday - Sunday).